### ISSUE TITLE
panic: runtime error: index out of range

### ISSUE THREAD

**Issue by moon86** [rodneymo/rig-monitorv2#44]
Hi,

I've got the following issue when I run the command ./grm -t :

```
gifter@vps552925:~/grm-2.2.1$ ./grm -t > trace.txt
panic: runtime error: index out of range
[trace.txt](https://github.com/rodneymo/rig-monitorv2/files/2108742/trace.txt)
[config.txt](https://github.com/rodneymo/rig-monitorv2/files/2108743/config.txt)




goroutine 13 [running]:
go-rig-monitor/miner.parseClaymoreOutputLine(0xc4206dbc50, 0xc420a12f20, 0xa6, 0xc4206dba00)
        /Users/ramf/go/src/go-rig-monitor/miner/claymore.go:384 +0x2834
go-rig-monitor/miner.(*RigConfig).LoadClaymmoreData(0xc420035ef8, 0xc420055ec0, 0x8e0e00)
        /Users/ramf/go/src/go-rig-monitor/miner/claymore.go:160 +0x63e
go-rig-monitor/miner.Monitor(0xc4201f61e0, 0xc420055ec0)
        /Users/ramf/go/src/go-rig-monitor/miner/rig-monitor.go:28 +0x182
created by main.main
        /Users/ramf/go/src/go-rig-monitor/main.go:147 +0xc37

```
I attach my config file (renamed in txt) and also the trace file.
Thanks for your help.

Regards

**Comment by rodneymo** [comment]
files are missing. Could you attach them?

**Comment by rodneymo** [comment]
From the trace I think I know what the issue is. Later today I'll try to reproduce it and create a patch.

**Comment by moon86** [comment]
Thank you.
While waiting for the update I created this little script that relaunch the program when it craches.

```
while true; do
echo "============ Starting Grafana ==============="
    if [ /home/username/grm-2.2.1/grm != "null" ]; then
        /home/username/grm-2.2.1/grm -t > /home/username/grm-2.2.1/grm.log
        echo "GRM ended/crashed. Restarting miner in 10 seconds --------------"
        sleep 10
    else
        echo "ERROR"
        sleep 10
    fi
done
```

**Comment by rodneymo** [comment]
I could not reproduce your error, but I noticed you are running an old version (2.2.1). There were some claymore issues fixed in 2.2.2. Could you try with the latest 2.3.1. version [https://github.com/rodneymo/rig-monitorv2/releases/tag/2.3.1](url)?

**Comment by moon86** [comment]
It seems better in 2.3.1. I'm going to monitor for a couple of hours before closing the issue.

```
============ Starting Grafana ===============
ERROR: 2018/06/17 13:52:23 claymore.go:157: Claymore paragraph did not contain GPU Specification lines
ERROR: 2018/06/17 13:57:24 claymore.go:157: Claymore paragraph did not contain GPU Specification lines
ERROR: 2018/06/17 14:13:22 claymore.go:157: Claymore paragraph did not contain GPU Specification lines

```

**Comment by moon86** [comment]
The issue is solved in the new version 2.3.1.
I close the issue.
Thank you for your help.
